java-processors/src/main/java/com/dataiku/dip/plugins/tradearea/GeoUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.dataiku.dip.plugins.tradearea;
+
+import com.dataiku.dip.shaker.types.GeoPoint;
+
+public class GeoUtils {
+
+    // radians/degrees conversion
```

Comment:
Field comments must be in the following format:
```suggestion
    /**
     * radians/degrees conversion
     */
```

Reply:
Ok